### PR TITLE
[admin] (Links) Updating reference anchor links

### PR DESCRIPTION
**Pull request by AlexJerabek**
[Staging site link](https://review.docs.microsoft.com/en-us/office/dev/add-ins/overview/office-add-ins?branch=AlexJ-ReferenceLinks)

This PR updates the links to our reference documentation to use the new link anchor format.

### CODE REVIEW

File: docs/reference/objectmodel/requirement-set-1.2/office.context.mailbox.md [hunk 1]
```diff
@@ -23,20 +23,20 @@ Provides access to the Outlook add-in object model for Microsoft Outlook.
...
+| [displayAppointmentForm(itemId)](/javascript/api/outlook/office.mailbox?view=outlook-js-1.2&preserve-view=true#outlook-office-mailbox-displayappointmentform-member(1)) | ReadItem | Compose<br>Read | [1.1](../requirement-set-1.1/outlook-requirement-set-1.1.md) |
+| [displayMessageForm(itemId)](/javascript/api/outlook/office.mailbox?view=outlook-js-1.2&preserve-view=true#outlook-office-mailbox-displaymessageform-member(1)) | ReadItem | Compose<br>Read | [1.1](../requirement-set-1.1/outlook-requirement-set-1.1.md) |
+| [displayNewAppointmentForm(parameters)](/javascript/api/outlook/office.mailbox?view=outlook-js-1.2&preserve-view=true#outlook-office-mailbox-displaynewappointmentform-member(1)) | ReadItem | Read | [1.1](../requirement-set-1.1/outlook-requirement-set-1.1.md) |
+| [getCallbackTokenAsync(callback, [userContext])](/javascript/api/outlook/office.mailbox?view=outlook-js-1.2&preserve-view=true#outlook-office-mailbox-getcallbacktokenasync-member(1)) | ReadItem | Compose<br>Read | [1.3](../requirement-set-1.3/outlook-requirement-set-1.3.md)<br>[1.1](../requirement-set-1.1/outlook-requirement-set-1.1.md) |
```
Comment (ElizabethSamuel-MSFT) [inline docs/reference/objectmodel/requirement-set-1.2/office.context.mailbox.md:40]:
```suggestion
| [getCallbackTokenAsync(callback, [userContext])](/javascript/api/outlook/office.mailbox?view=outlook-js-1.2&preserve-view=true#outlook-office-mailbox-getcallbacktokenasync-member(1)) | ReadItem | Read | [1.1](../requirement-set-1.1/outlook-requirement-set-1.1.md) |
```

File: docs/reference/objectmodel/requirement-set-1.5/office.context.mailbox.md [hunk 2]
```diff
@@ -23,33 +23,33 @@ Provides access to the Outlook add-in object model for Microsoft Outlook.
...
+| [displayMessageForm(itemId)](/javascript/api/outlook/office.mailbox?view=outlook-js-1.5&preserve-view=true#outlook-office-mailbox-displaymessageform-member(1)) | ReadItem | Compose<br>Read | [1.1](../requirement-set-1.1/outlook-requirement-set-1.1.md) |
+| [displayNewAppointmentForm(parameters)](/javascript/api/outlook/office.mailbox?view=outlook-js-1.5&preserve-view=true#outlook-office-mailbox-displaynewappointmentform-member(1)) | ReadItem | Read | [1.1](../requirement-set-1.1/outlook-requirement-set-1.1.md) |
+| [getCallbackTokenAsync([options], callback)](/javascript/api/outlook/office.mailbox?view=outlook-js-1.5&preserve-view=true#outlook-office-mailbox-getcallbacktokenasync-member(1)) | ReadItem | Compose<br>Read | [1.5](../requirement-set-1.5/outlook-requirement-set-1.5.md) |
+| [getCallbackTokenAsync(callback, [userContext])](/javascript/api/outlook/office.mailbox?view=outlook-js-1.5&preserve-view=true#outlook-office-mailbox-getcallbacktokenasync-member(1)) | ReadItem | Compose<br>Read | [1.3](../requirement-set-1.3/outlook-requirement-set-1.3.md)<br>[1.1](../requirement-set-1.1/outlook-requirement-set-1.1.md) |
```
Comment (ElizabethSamuel-MSFT) [inline docs/reference/objectmodel/requirement-set-1.5/office.context.mailbox.md:45]:
```suggestion
| [getCallbackTokenAsync(callback, [userContext])](/javascript/api/outlook/office.mailbox?view=outlook-js-1.5&preserve-view=true#outlook-office-mailbox-getcallbacktokenasync-member(2)) | ReadItem | Compose<br>Read | [1.3](../requirement-set-1.3/outlook-requirement-set-1.3.md)<br>[1.1](../requirement-set-1.1/outlook-requirement-set-1.1.md) |
```

File: docs/reference/objectmodel/requirement-set-1.6/office.context.mailbox.md [hunk 3]
```diff
@@ -23,34 +23,34 @@ Provides access to the Outlook add-in object model for Microsoft Outlook.
...
+| [displayNewAppointmentForm(parameters)](/javascript/api/outlook/office.mailbox?view=outlook-js-1.6&preserve-view=true#outlook-office-mailbox-displaynewappointmentform-member(1)) | ReadItem | Read | [1.1](../requirement-set-1.1/outlook-requirement-set-1.1.md) |
+| [displayNewMessageForm(parameters)](/javascript/api/outlook/office.mailbox?view=outlook-js-1.6&preserve-view=true#outlook-office-mailbox-displaynewmessageform-member(1)) | ReadItem | Read | [1.6](../requirement-set-1.6/outlook-requirement-set-1.6.md) |
+| [getCallbackTokenAsync([options], callback)](/javascript/api/outlook/office.mailbox?view=outlook-js-1.6&preserve-view=true#outlook-office-mailbox-getcallbacktokenasync-member(1)) | ReadItem | Compose<br>Read | [1.5](../requirement-set-1.5/outlook-requirement-set-1.5.md) |
+| [getCallbackTokenAsync(callback, [userContext])](/javascript/api/outlook/office.mailbox?view=outlook-js-1.6&preserve-view=true#outlook-office-mailbox-getcallbacktokenasync-member(1)) | ReadItem | Compose<br>Read | [1.3](../requirement-set-1.3/outlook-requirement-set-1.3.md)<br>[1.1](../requirement-set-1.1/outlook-requirement-set-1.1.md) |
```
Comment (ElizabethSamuel-MSFT) [inline docs/reference/objectmodel/requirement-set-1.6/office.context.mailbox.md:46]:
```suggestion
| [getCallbackTokenAsync(callback, [userContext])](/javascript/api/outlook/office.mailbox?view=outlook-js-1.6&preserve-view=true#outlook-office-mailbox-getcallbacktokenasync-member(2)) | ReadItem | Compose<br>Read | [1.3](../requirement-set-1.3/outlook-requirement-set-1.3.md)<br>[1.1](../requirement-set-1.1/outlook-requirement-set-1.1.md) |
```

File: docs/reference/objectmodel/requirement-set-1.7/office.context.mailbox.md [hunk 4]
```diff
@@ -23,34 +23,34 @@ Provides access to the Outlook add-in object model for Microsoft Outlook.
...
+| [displayNewAppointmentForm(parameters)](/javascript/api/outlook/office.mailbox?view=outlook-js-1.7&preserve-view=true#outlook-office-mailbox-displaynewappointmentform-member(1)) | ReadItem | Read | [1.1](../requirement-set-1.1/outlook-requirement-set-1.1.md) |
+| [displayNewMessageForm(parameters)](/javascript/api/outlook/office.mailbox?view=outlook-js-1.7&preserve-view=true#outlook-office-mailbox-displaynewmessageform-member(1)) | ReadItem | Read | [1.6](../requirement-set-1.6/outlook-requirement-set-1.6.md) |
+| [getCallbackTokenAsync([options], callback)](/javascript/api/outlook/office.mailbox?view=outlook-js-1.7&preserve-view=true#outlook-office-mailbox-getcallbacktokenasync-member(1)) | ReadItem | Compose<br>Read | [1.5](../requirement-set-1.5/outlook-requirement-set-1.5.md) |
+| [getCallbackTokenAsync(callback, [userContext])](/javascript/api/outlook/office.mailbox?view=outlook-js-1.7&preserve-view=true#outlook-office-mailbox-getcallbacktokenasync-member(1)) | ReadItem | Compose<br>Read | [1.3](../requirement-set-1.3/outlook-requirement-set-1.3.md)<br>[1.1](../requirement-set-1.1/outlook-requirement-set-1.1.md) |
```
Comment (ElizabethSamuel-MSFT) [inline docs/reference/objectmodel/requirement-set-1.7/office.context.mailbox.md:46]:
```suggestion
| [getCallbackTokenAsync(callback, [userContext])](/javascript/api/outlook/office.mailbox?view=outlook-js-1.7&preserve-view=true#outlook-office-mailbox-getcallbacktokenasync-member(2)) | ReadItem | Compose<br>Read | [1.3](../requirement-set-1.3/outlook-requirement-set-1.3.md)<br>[1.1](../requirement-set-1.1/outlook-requirement-set-1.1.md) |
```

File: docs/reference/objectmodel/requirement-set-1.8/office.context.mailbox.md [hunk 5]
```diff
@@ -23,35 +23,35 @@ Provides access to the Outlook add-in object model for Microsoft Outlook.
...
+| [displayNewAppointmentForm(parameters)](/javascript/api/outlook/office.mailbox?view=outlook-js-1.8&preserve-view=true#outlook-office-mailbox-displaynewappointmentform-member(1)) | ReadItem | Read | [1.1](../requirement-set-1.1/outlook-requirement-set-1.1.md) |
+| [displayNewMessageForm(parameters)](/javascript/api/outlook/office.mailbox?view=outlook-js-1.8&preserve-view=true#outlook-office-mailbox-displaynewmessageform-member(1)) | ReadItem | Read | [1.6](../requirement-set-1.6/outlook-requirement-set-1.6.md) |
+| [getCallbackTokenAsync([options], callback)](/javascript/api/outlook/office.mailbox?view=outlook-js-1.8&preserve-view=true#outlook-office-mailbox-getcallbacktokenasync-member(1)) | ReadItem | Compose<br>Read | [1.5](../requirement-set-1.5/outlook-requirement-set-1.5.md) |
+| [getCallbackTokenAsync(callback, [userContext])](/javascript/api/outlook/office.mailbox?view=outlook-js-1.8&preserve-view=true#outlook-office-mailbox-getcallbacktokenasync-member(1)) | ReadItem | Compose<br>Read | [1.3](../requirement-set-1.3/outlook-requirement-set-1.3.md)<br>[1.1](../requirement-set-1.1/outlook-requirement-set-1.1.md) |
```
Comment (ElizabethSamuel-MSFT) [inline docs/reference/objectmodel/requirement-set-1.8/office.context.mailbox.md:47]:
```suggestion
| [getCallbackTokenAsync(callback, [userContext])](/javascript/api/outlook/office.mailbox?view=outlook-js-1.8&preserve-view=true#outlook-office-mailbox-getcallbacktokenasync-member(2)) | ReadItem | Compose<br>Read | [1.3](../requirement-set-1.3/outlook-requirement-set-1.3.md)<br>[1.1](../requirement-set-1.1/outlook-requirement-set-1.1.md) |
```

File: docs/reference/objectmodel/requirement-set-1.9/office.context.mailbox.md [hunk 6]
```diff
@@ -23,39 +23,39 @@ Provides access to the Outlook add-in object model for Microsoft Outlook.
...
+| [displayNewMessageForm(parameters)](/javascript/api/outlook/office.mailbox?view=outlook-js-1.9&preserve-view=true#outlook-office-mailbox-displaynewmessageform-member(1)) | ReadItem | Read | [1.6](../requirement-set-1.6/outlook-requirement-set-1.6.md) |
+| [displayNewMessageFormAsync(parameters, [options], [callback])](/javascript/api/outlook/office.mailbox?view=outlook-js-1.9&preserve-view=true#outlook-office-mailbox-displaynewmessageformasync-member(1)) | ReadItem | Read | [1.9](outlook-requirement-set-1.9.md) |
+| [getCallbackTokenAsync([options], callback)](/javascript/api/outlook/office.mailbox?view=outlook-js-1.9&preserve-view=true#outlook-office-mailbox-getcallbacktokenasync-member(1)) | ReadItem | Compose<br>Read | [1.5](../requirement-set-1.5/outlook-requirement-set-1.5.md) |
+| [getCallbackTokenAsync(callback, [userContext])](/javascript/api/outlook/office.mailbox?view=outlook-js-1.9&preserve-view=true#outlook-office-mailbox-getcallbacktokenasync-member(1)) | ReadItem | Compose<br>Read | [1.3](../requirement-set-1.3/outlook-requirement-set-1.3.md)<br>[1.1](../requirement-set-1.1/outlook-requirement-set-1.1.md) |
```
Comment (ElizabethSamuel-MSFT) [inline docs/reference/objectmodel/requirement-set-1.9/office.context.mailbox.md]:
```suggestion
| [getCallbackTokenAsync(callback, [userContext])](/javascript/api/outlook/office.mailbox?view=outlook-js-1.9&preserve-view=true#outlook-office-mailbox-getcallbacktokenasync-member(2)) | ReadItem | Compose<br>Read | [1.3](../requirement-set-1.3/outlook-requirement-set-1.3.md)<br>[1.1](../requirement-set-1.1/outlook-requirement-set-1.1.md) |
```

File: docs/outlook/authentication.md [hunk 7]
```diff
@@ -45,15 +45,15 @@ Exchange user identity tokens provide a way for your add-in to establish the ide
 - When the add-in needs access to a non-Microsoft service that you control.
 - As a fallback authentication when the add-in is running on a version of Office that doesn't support SSO.
 
-Your add-in can call [getUserIdentityTokenAsync](/javascript/api/outlook/office.mailbox#getCallbackTokenAsync_callback__userContext_) to get Exchange user identity tokens. For details on using these tokens, see [Authenticate a user with an identity token for Exchange](authenticate-a-user-with-an-identity-token.md).
+Your add-in can call [getUserIdentityTokenAsync](/javascript/api/outlook/office.mailbox#outlook-office-mailbox-getcallbacktokenasync-member(1)) to get Exchange user identity tokens. For details on using these tokens, see [Authenticate a user with an identity token for Exchange](authenticate-a-user-with-an-identity-token.md).
```
Comment (ElizabethSamuel-MSFT) [inline docs/outlook/authentication.md:48]:
```suggestion
Your add-in can call [getUserIdentityTokenAsync](/javascript/api/outlook/office.mailbox#outlook-office-mailbox-getuseridentitytokenasync-member(1)) to get Exchange user identity tokens. For details on using these tokens, see [Authenticate a user with an identity token for Exchange](authenticate-a-user-with-an-identity-token.md).
```

**Review by ElizabethSamuel-MSFT — CHANGES_REQUESTED** [review]
Found a few missed ones. Also took the opportunity to include some minor cleanup.

**Comment by AlexJerabek** [comment]
Thanks for taking a look @ElizabethSamuel-MSFT. I believe I either applied all your suggestions or fixed them with another regex pass. Please take another look at the recent commits when you can.

File: docs/reference/requirement-sets/excel-api-1-13-requirement-set.md [hunk 8]
```diff
@@ -12,9 +12,9 @@ The ExcelApi 1.13 added a method to insert worksheets into a workbook from a Bas
...
+| [Formula changed events](../../excel/excel-add-ins-worksheets.md#detect-formula-changes) | Track changes to formulas, including the source and type of event that caused a change. | [Worksheet.onFormulaChanged](/javascript/api/excel/excel.worksheet#excel-excel-worksheet-onformulachanged-member)|
+| [Formula dependents](../../excel/excel-add-ins-ranges-precedents-dependents.md#get-the-direct-dependents-of-a-formula) | Locate the direct dependent cells of a formula. | [Range.getDirectDependents](/javascript/api/excel/excel.range#excel-excel-range-getdirectdependents-member(1)) |
+| [Insert worksheets](../../excel//excel-add-ins-workbooks.md#insert-a-copy-of-an-existing-workbook-into-the-current-one) | Insert worksheets from another workbook into the current workbook as a Base64-encoded string. | [Workbook.insertWorksheetsFromBase64](/javascript/api/excel/excel.workbook#excel-excel-workbook-insertworksheetsfrombase64-member(1)) |
```
Comment (ElizabethSamuel-MSFT) [inline docs/reference/requirement-sets/excel-api-1-13-requirement-set.md]:
```suggestion
| [Insert worksheets](../../excel/excel-add-ins-workbooks.md#insert-a-copy-of-an-existing-workbook-into-the-current-one) | Insert worksheets from another workbook into the current workbook as a Base64-encoded string. | [Workbook.insertWorksheetsFromBase64](/javascript/api/excel/excel.workbook#excel-excel-workbook-insertworksheetsfrombase64-member(1)) |
```

**Review by ElizabethSamuel-MSFT — APPROVED** [review]

File: docs/reference/requirement-sets/excel-api-1-13-requirement-set.md (hunk 8 above)
Comment (AlexJerabek) [inline docs/reference/requirement-sets/excel-api-1-13-requirement-set.md]:
```suggestion
| [Insert worksheets](../../excel/excel-add-ins-workbooks.md#insert-a-copy-of-an-existing-workbook-into-the-current-one) | Insert worksheets from another workbook into the current workbook as a Base64-encoded string. | [Workbook.insertWorksheetsFromBase64](/javascript/api/excel/excel.workbook#excel-excel-workbook-insertworksheetsfrombase64-member(1)) |
```

**Pull request merged**